src/sentry/ownership/grammar.py

```diff
@@ -563,8 +571,9 @@ def create_schema_from_issue_owners(
     try:
         rules = parse_rules(issue_owners)
     except ParseError as e:
+        rule_name = e.expr.name if e.expr else str(e.expr)
         raise ValidationError(
-            {"raw": f"Parse error: {e.expr.name} (line {e.line()}, column {e.column()})"}
+            {"raw": f"Parse error: {rule_name} (line {e.line()}, column {e.column()})"}
```

Comment:
This could probably just be `{"raw": str(e)}` but I wanted to keep the message the same for now.

`rule_name` matches the internal behavior now: https://github.com/erikrose/parsimonious/blob/master/parsimonious/exceptions.py#L23-L24

Reply:
this is going to make a weird `Parse error: None (...)` -- just making sure that's what you intend

Reply:
Given it previously would have raised an AttributeError I suspect it's unreachable but the line/column info will be enough to debug if it ever does happen